packages/fxa-auth-server/lib/routes/subscriptions.js

```diff
@@ -915,7 +988,9 @@ const createRoutes = (
       },
       handler: async function(request) {
         log.begin('subscriptions.getClients', request);
-        return CLIENT_CAPABILITIES;
+        return Object.entries(
```

Comment:
This code should probably just be removed along with the rest of the subhub-related routes, but figured I'd move it here to get it out of the way. It would technically still work if we reverted to subhub usage